### PR TITLE
Change link from ocsigen-start to ocsigen-start/1.1.0/manual/intro

### PR DESCRIPTION
**Pull request by CelieArnaud**
in file _post/2017-02-09-start.md
+ try a fix for redirection of Os-demo in the same file

### CODE REVIEW

File: _posts/2017-02-09-start.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ Ocsigen Start and Ocsigen Toolkit are developed by the Ocsigen team at
 [Be Sport][besport], at the [Paris Diderot University][p7], and at the
 [IRILL][irill].
 
-[os]:         https://ocsigen.org/ocsigen-start/
+[os]:         http://ocsigen.org/ocsigen-start/1.1.0/manual/intro
 [ot]:         https://ocsigen.org/ocsigen-toolkit/
-[osdemo]:     http://ocsigen.org/ocsigen-start/demo/
+[osdemo]:     http:///ocsigen.org/ocsigen-start/demo/
```
Comment (vasilisp) [inline _posts/2017-02-09-start.md]: Probably a typo?

Comment (CelieArnaud) [reply _posts/2017-02-09-start.md]: Yes, I'll fix that

**Comment by vasilisp** [comment]
Thanks! Actually, you have added symlinks and we have fixed HTTPS, so https://ocsigen.org/ocsigen-start/ works :). Shouldn't we just keep the shorter URL?

**Comment by CelieArnaud** [comment]
I don't think, because https://ocsigen.org/ocsigen-start/ and https://ocsigen.org/ocsigen-start/demo/ have not the same content. https://ocsigen.org/ocsigen-start/demo/ should redirect to http://ocsigen-1.inria.fr/ocsigen-start/demo/ unlike https://ocsigen.org/ocsigen-start/

**Comment by vasilisp** [comment]
No objection, but we need to fix the `[osdemo]` URL in that case, not `[os]`. `[os]` refers to the project and points to an existing URL.

**Pull request merged**